OpenRA.Game/Network/UnitOrders.cs

```diff
@@ -18,6 +18,8 @@ namespace OpenRA.Network
 {
 	static class UnitOrders
 	{
+		private const string ServerChatName = "Battlefield Control";
```

Comment:
You should drop the explicit `private`. Also private variables should be `camelCase`. (See https://github.com/OpenRA/OpenRA/wiki/Coding-Standard#the-short-version . Maybe we want to make this `public` instead then?)
Also, do we want to have a `[Translate]` tag attached to this?